tests/cpu_avr8_xmega_drivers/Makefile

```
@@ -4,6 +4,7 @@ include ../Makefile.tests_common
 
 # list of ATxmega boards
 BOARD_WHITELIST = \
+  atxmega-a1-xplained \
   atxmega-a1u-xpro \
   atxmega-a3bu-xplained \
   #
```

You could also do
```suggestion
FEATURES_REQUIRED += cpu_atxmega
```

but this can be a follow-up